app/helpers/audits_helper.rb

```diff
@@ -13,7 +13,7 @@ def id_to_label(name, change)
       when /.*_id$/
         name.classify.gsub('Id','').constantize.find(change).to_label
       else
-        change.to_s
+        change.to_s.include?(EncryptValue::ENCRYPTION_PREFIX) ? _("[encrypted]") : change.to_s
```

Comment:
Is it possible to move this into the Audit model (via AuditExtensions) at all? The encrypted value will remain visible via the API and is of course, gets needlessly stored.